service/src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -574,7 +577,19 @@ private AzureProfile getAzureProfile(AzureCloudContext azureCloudContext) {
     return new AzureProfile(
         azureCloudContext.getAzureTenantId(),
         azureCloudContext.getAzureSubscriptionId(),
-        AzureEnvironment.AZURE);
+        getAzureEnvironmentFromName(azureConfiguration.getAzureEnvironment()));
+  }
+
+  public AzureEnvironment getAzureEnvironmentFromName(String envName) {
```

Comment:
The JDK differs from the .NET SDK in that there is no 'name' attribute that we can interrogate and match on. Hard coding the 3 available AzureEnvironments in this switch statement instead